docs/HowTo/Store-Keys/Use-Hashicorp.md

```diff
@@ -54,15 +52,32 @@ option set to `8590` to avoid conflict with the default EthSigner listening port
     protection. That is, the genesis file must include at least the Spurious Dragon milestone
     (defined as `eip158Block` in the genesis file) so the blockchain is using a chain ID.
 
+## Create the Known Servers File
```

Comment:
Given TLS is enabled by default, is it a requirement to create this file unless you explicitly disable TLS?  ie, what happens if you don't create it and haven't explicitly disabled TLS? 

If it's a requirement to create this file when TLS is on by default, it probably makes sense to move the sentence from the following section about how to disable TLS to this section.  And also make it clear creating this file is required unless you explicitly disable TLS.

Reply:
Done.